inductiva/resources/utils.py

```diff
@@ -0,0 +1,20 @@
+"""Methods related to managing executers and resource pools."""
+import uuid
+
+import inductiva
+
+
+def create_machine_group_id() -> uuid.UUID:
+    """Request the creation of a machine group ID."""
+
+    api_config = inductiva.api.validate_api_key(inductiva.api_key)
+
+    with inductiva.client.ApiClient(api_config) as client:
+        api_instance = inductiva.client.apis.tags.executers_api.ExecutersApi(client)
+
+        try:
+            api_response = api_instance.create_resource_pool()
```

Comment:
I don't think this makes sense. I mean, I may be understanding something wrong, but what is create_resource_pool doing? Is it an empty resource pool? where are the parameters of the machines, etc..

Reply:
This will also be passed to the backend... but at the moment, it asks the backend to create a single string that will be used to unique identify the machine group in the DB.